cmd/security-agent/subcommands/runtime/command_unsupported.go

```diff
@@ -28,7 +29,7 @@ func Commands(*command.GlobalParams) []*cobra.Command {
 }
 
 // StartRuntimeSecurity starts runtime security
-func StartRuntimeSecurity(log log.Component, config config.Component, _ string, _ startstop.Stopper, _ statsd.ClientInterface, _ workloadmeta.Component) (*secagent.RuntimeSecurityAgent, error) {
+func StartRuntimeSecurity(log log.Component, config config.Component, _ string, _ startstop.Stopper, _ statsd.ClientInterface, _ workloadmeta.Component, _ compression.Component) (*secagent.RuntimeSecurityAgent, error) {
```

Comment:
You need to have `_ YOUR_COMPONENT` only when the component doesn't expose any methods or when it listens to a port. I don't think that the case for `compression.Component`.  What does happen if you remove  ` _ compression.Component`?